dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java

```diff
@@ -81,4 +81,10 @@ public void testDateWithExtraLeadingWhiteSpace() throws Exception {
         Assert.assertEquals(1175052759000L, f.getDate().getTime());
     }
 
+    @Test
+    public void testDateWhenNoTimezone() throws Exception {
+        DateTimeField f = parse("Date: Fri, 05 Jan 2018 16:18:28 Z");
```

Comment:
Z equals UTC and it's definitely a timezone, it's called Zulu Timezone (no joke)

Reply:
Fixed, and in a valid way. CF https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html

Reply:
maybe you also need to rename this test then